maven-scm-providers/maven-scm-providers-cvs/maven-scm-provider-cvs-commons/src/test/java/org/apache/maven/scm/provider/cvslib/command/changelog/CvsChangeLogConsumerTest.java

```diff
@@ -71,8 +71,8 @@ public void testParse()
         for ( Iterator<ChangeSet> i = entries.iterator(); i.hasNext(); )
         {
             entry = i.next();
-            assertTrue( "ChangeLogEntry erroneously picked up",
-                        entry.toString().indexOf( "ChangeLogEntry.java" ) == -1 );
+            assertEquals( "ChangeLogEntry erroneously picked up",
```

Comment:
The purpose of this test is contains, and not the -1 whi h is a technical detail.

Reply:
Revert it